Simplified/AxisService/NYPLAxisPackagePathPrefixProvider.swift

```diff
@@ -37,6 +37,10 @@ class NYPLAxisPackagePathPrefixProvider: NYPLAxisPackagePathPrefixProviding {
       .deletingLastPathComponent()
```

Comment:
I didn't notice this in previous reviews, but this function is called `getXYZ` which kind of implies a function with no side effects, however at line 36 we are changing the value of the `packagePathPrefix` property. It would be make it for clearer semantics IMO to rename this function to something like `calculateXYZ` or `generateXYZ`.

Reply:
You're right. Changed it to a struct since we're only using it once per download session and therefore do not need to cache the value of packagePathPrefix